docs/developer-docs/deploy/computation-and-storage-costs.md

```diff
@@ -10,45 +10,49 @@ The Internet Computer is a decentralized public utility, controlled by the NNS 
 
 Canister smart contract computations running on the Internet Computer blockchain are fueled by “cycles”, which play a similar role to “gas” on Ethereum. There are several major differences however. One of the most fundamental differences is that Ethereum leverages “user pays” and the Internet Computer and “smart contract pays” (sometimes called “reverse gas”) model. Whereas the Ethereum blockchain requires end users to send payments for the gas smart contracts consume with every transaction, on the Internet Computer, Canister smart contracts are pre-charged with cycles, such that contracts effectively pay for their own computation - freeing users from the responsibility.
 
-See below for details on the cost of compute and storage transactions on the Internet Computer as of August 29, 2022.
+In late November 2022, high-replication application subnets have been made available on the Internet Computer. The first such subnets launch with a replication factor of 34, different sizes may be available in the future. The pricing is determined by linearly scaling the prices for 13-node subnets to the new subnet size. The below tables provide pricing in cycles and USD for the new 34-node subnets. The linear scaling for a transaction is computed using the following formula, where *n* is the size of the subnet to compute the price for, *13_node_price* is the price for the transaction on the reference-size subnet with 13 nodes, and *DIV* is integer division:
```

Comment:
> In late November 2022, high-replication application subnets have been made available on the Internet Computer.

Well, technically, the subnet will be created late November but it'll probably be early December before it's made available to people for actual canister deployment (need to enable cost scaling on the replica by turning the flag on and also submit 2 proposals to make the subnet available to public as a high replication subnet).

> The pricing is determined by linearly scaling the prices for 13-node subnets to the new subnet size.

Slightly better read imo: Cycles prices for the new 34-node subnets are scaled linearly based on the number of nodes from the base prices for 13-node subnets.

Reply:
OK, updated this.